.github/scripts/test_open_data_assets.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python3
+# Run tests on our open data assets to confirm that their row counts match
+# the row counts of the underlying Athena tables when grouped by year.
+#
+# Requires one positional argument, the path to a dbt manifest that can be
+# used to parse exposure data representing open data assets and their
+# dependency graph.
+#
+# Example:
+#
+#   cd dbt
+#   python3 ../.github/scripts/test_open_data_assets.py target/manifest.json
+
+import contextlib
+import datetime
+import io
+import json
+import sys
+import typing
+
+import agate
+import requests
+from dbt.cli.main import dbtRunner
+
+# API URL and params that we can use to grab counts by year for each asset
+ASSET_API_URL = "https://datacatalog.cookcountyil.gov/resource/{asset_id}.json"
+ASSET_API_QUERY_PARAMS = {"$query": "SELECT COUNT(*),year GROUP BY year"}
+DBT = dbtRunner()
```

Comment:
TIL that dbt has an [interface](https://docs.getdbt.com/reference/programmatic-invocations) for running CLI commands from Python scripts!